build.gradle.kts

```diff
@@ -1,101 +1,45 @@
-    kotlin("plugin.spring") version "1.3.71"
-
-    // Adds a no-arg (Java) constructor to classes annotated with @Entity, @Embeddable or @MappedSuperclass
-    kotlin("plugin.jpa") version "1.3.71"
-
-    id("com.github.ben-manes.versions") version "0.28.0"
-    id("org.owasp.dependencycheck") version "5.3.2.1"
-}
-
-repositories {
-    mavenLocal()
-    mavenCentral()
-}
-
-group = "uk.gov.justice.digital.hmpps"
-
-val today: Instant = Instant.now()
-val todaysDate: String = LocalDate.now().format(ISO_DATE)
-
-version = if (System.getenv().containsKey("CI")) {
-    "${todaysDate}.${System.getenv("CIRCLE_BUILD_NUM")}"
-} else {
-    todaysDate
-}
-
-springBoot {
-    buildInfo {
-        properties {
-            artifact = rootProject.name
-            version = version
-            group = group
-            name = rootProject.name
-            time = today
-
-            additional = mapOf(
-                    "by" to System.getProperty("user.name"),
-                    "operatingSystem" to "${System.getProperty("os.name")} (${System.getProperty("os.version")})",
-                    "continuousIntegration" to System.getenv().containsKey("CI"),
-                    "machine" to InetAddress.getLocalHost().hostName
-            )
-        }
-    }
+    id("uk.gov.justice.hmpps.gradle-spring-boot") version "0.3.0"
+    kotlin("plugin.spring") version "1.3.72"
+    kotlin("plugin.jpa") version "1.3.72"
 }
 
 configurations {
-
-    implementation {
-        exclude(mapOf("module" to "tomcat-jdbc"))
-    }
+    implementation { exclude(mapOf("module" to "tomcat-jdbc")) }
 }
 
+ext ["spring-security.version"] = "5.3.1.RELEASE"
+
 dependencies {
 
     annotationProcessor("org.springframework.boot:spring-boot-configuration-processor")
 
     runtimeOnly("com.h2database:h2:1.4.200")
-    runtimeOnly("org.flywaydb:flyway-core:6.3.3")
+    runtimeOnly("org.flywaydb:flyway-core:6.4.1")
     runtimeOnly("org.postgresql:postgresql:42.2.12")
 
     implementation(kotlin("stdlib-jdk8"))
     implementation(kotlin("reflect"))
 
-    implementation("org.springframework.boot:spring-boot-starter-web")
     implementation("org.hibernate:hibernate-java8")
-    implementation("org.springframework.boot:spring-boot-starter-actuator")
     implementation("org.springframework.boot:spring-boot-starter-data-jpa")
     implementation("org.springframework.boot:spring-boot-starter-cache")
     implementation("org.springframework.boot:spring-boot-starter-security")
-    implementation("org.springframework.security.oauth:spring-security-oauth2:2.4.0.RELEASE")
-    implementation("org.springframework.security:spring-security-jwt:1.1.0.RELEASE")
+    implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
+    implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
+
+    implementation("org.springframework.security:spring-security-oauth2-jose")
+
     implementation("net.logstash.logback:logstash-logback-encoder:6.3")
```

Comment:
don't need `logback` as included in plugin

Reply:
removed